packages/client/src/components/app/forms/RelationshipField.svelte

```diff
@@ -1,17 +1,23 @@
 <script lang="ts">
   import { CoreSelect, CoreMultiselect } from "@budibase/bbui"
-  import { FieldType } from "@budibase/types"
+  import {
+    FieldType,
+    GroupUserDatasource,
+    InternalTable,
+    TableDatasource,
+    UserDatasource,
+  } from "@budibase/types"
   import { fetchData, Utils } from "@budibase/frontend-core"
   import { getContext } from "svelte"
   import Field from "./Field.svelte"
   import type {
     SearchFilter,
     RelationshipFieldMetadata,
-    Table,
     Row,
   } from "@budibase/types"
-
-  const { API } = getContext("sdk")
+  import TableFetch from "@budibase/frontend-core/src/fetch/TableFetch"
```

Comment:
This should not be needed if this one gets applied: https://github.com/Budibase/budibase/pull/15650/commits/fa2357c1d34ead3a5ef7b628775d94bdec6bc2d5